### DISCUSSION TITLE
How can merger pull-request?

### DISCUSSION THREAD

**Discussion by HappyMicky0317** [community/community#58109] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Someone made pull-request for my repo, but I can't find it and merge. How can I do it?

**Comment by jge162** [comment, marked as answer]
should be an open pull request under the PR tab.

**Reply by ghostinhershell** [reply]
Hey @HappyDream0317, 

@jge162's answer here is the correct answer. I'm just going to link further reading from our GitHub Docs for you to reference as well. Let us know if you have any additional questions, good luck! 🚀

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HappyDream0317! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
